lib/screens/login_screen.dart

```diff
@@ -1,42 +1,85 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_excellence_hr/resources/app_colors.dart';
-import '../screens/login_bottom.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
```

Comment:
we have two screens? login screen and login_page. i dont understand why two screens for login?

Reply:
ok. so this structure is wrong.
as u have kept both of the file in screens/ folder. this means two actual different screens for the user.

we should have /screens/login.dart
and then can we have /widgets/login/.....  etc inside this all widgets for login

Reply:
ok will change

Reply:
@manishiitg done